packages/editor-ui/src/components/RunData.vue

```diff
@@ -243,21 +243,11 @@
 				<RunDataTable :node="node" :inputData="inputData" :mappingEnabled="mappingEnabled" :distanceFromActive="distanceFromActive" :showMappingHint="showMappingHint" :runIndex="runIndex" :totalRuns="maxRunIndex" @mounted="$emit('tableMounted', $event)" />
 			</div>
 
-			<div v-else-if="hasNodeRun && displayMode === 'json'" :class="$style.jsonDisplay">
-				<vue-json-pretty
-					:data="jsonData"
-					:deep="10"
-					v-model="selectedOutput.path"
-					:showLine="true"
-					:showLength="true"
-					selectableType="single"
-					path=""
-					:highlightSelectedNode="true"
-					:selectOnClickNode="true"
-					@click="dataItemClicked"
-					class="json-data"
-				/>
-			</div>
+			<run-data-json
+				v-else-if="hasNodeRun && displayMode === 'json'"
+				:jsonData="jsonData"
```

Comment:
for completeness, you should move jsonData computed function into `run-data-json`, pass in `inputData` as prop similar to table view and make convertToJSON a helper function..